### ISSUE TITLE
Ctrl+. not working

### ISSUE THREAD

**Issue by Skeftomilos** [MicrosoftDocs/visualstudio-docs#5187]
I just upgraded to Microsoft Visual Studio Community 2019 Version 16.5.4, and the Ctrl+. shortcut is not working any more. The Rename option does not appear among the other refactoring suggestions.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d231e9f6-7c0c-7073-6727-81a08019e589
* Version Independent ID: d0078914-0fc3-e988-ce0e-559c09f0b4c8
* Content: [Refactor rename - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/reference/rename?view=vs-2019#feedback)
* Content Source: [docs/ide/reference/rename.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/ide/reference/rename.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-general**
* GitHub Login: @TerryGLee
* Microsoft Alias: **tglee**

**Comment by msebolt** [comment]
@Skeftomilos Thank you for the feedback, assigning this to the author for review.

Please also consider filing this with the product team [here](https://developercommunity.visualstudio.com/spaces/21/index.html) or let us know and we start a ticket for you.

**Comment by TerryGLee** [comment]
@Skeftomilos Thanks for contacting us. Unfortunately, I was not able to repo what you reported in Microsoft Visual Studio Community 2019 Version 16.5.4. It might be something specific to your configuration or it might be a bug; either way, the best course forward is to alert the product team of the problem, [here](https://developercommunity.visualstudio.com/spaces/21/index.html).

Meanwhile, to make the content more clear on what one should expect when they press **Ctrl**+**.** or expand the lightbulb icon menu, I've added the following screenshot to it. The updated page should be published on 5/5/2020.

![image](https://user-images.githubusercontent.com/23244653/81026083-c485d880-8e2d-11ea-9d75-459fc4da29db.png)

Thanks for your help in improving our product and our documentation. 

Cheers,
~ Terry

**Comment by Skeftomilos** [comment]
Hi @TerryGLee, thanks for looking at the issue. I finally managed to figure out what is different with my configuration. I had disabled an important analyzer in my .NET Core 3.1 Console app project, by unchecking the [Properties > Code Analysis > Analyzers > Run on live analysis] checkbox. After enabling again this option I was able to use the Ctrl+. shortcut for renaming identifiers in my project as usual. Apparently between the VS versions 16.3.9 and 16.5.4 a change happened that connected the setting `RunAnalyzersDuringLiveAnalysis` with the "Rename" refactoring ability of the screwdriver icon.

Here is a screenshot of what is displayed now with Ctrl+. and the aforementioned option disabled, after renaming the class MyClass to Person.

![Screenshot](https://user-images.githubusercontent.com/48834899/81470502-bb14bb80-91f3-11ea-83d9-b9a6b1f9e2de.png)
